src/explore-education-statistics-admin/src/components/editable/Comments.tsx

```diff
@@ -60,8 +62,10 @@ const Comments = ({
         blockId,
         additionalComment,
       );
+      toggleSubmitting(true);
```

Comment:
Would put this before the `if`. Toggling here won't stop a user being able to potentially click the submit multiple times (adding multiples of the same comment).